<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignalGenerator</class>
 <widget class="QWidget" name="SignalGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>573</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Scopy Signal Generator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* QComboBoxes*/
QComboBox
{
  height: 24px;
  border: 0px;
  color: white;
  font-size: 14px;
  border-bottom: 1px solid rgba(255, 255, 255, 102);
  padding-bottom: 4px;
}
QComboBox::drop-down
{
  subcontrol-position: center right;
  width: 10px;
  height: 6px;
  border-image: url(:/icons/sba_cmb_box_arrow.svg);
}
QComboBox QAbstractItemView {
  border: 0px;
  color: white;
  selection-background-color: rgba(25, 25, 25);
  outline: 0px;
}
QComboBox:disabled
{
  color: rgb(85, 85, 85);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="run_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: white;
background-color: orange;
font-family: &quot;monospace&quot;;
font-weight: bold;
height: 30px;
border-width: 0px;
border-radius: 3px;
min-width: 6em;
}
QPushButton:disabled {
background-color: grey;
}
QPushButton:checked {
background-color: #F45000;
}</string>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
       <property name="icon">
        <iconset theme="media-playback-start">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="plot"/>
     </item>
     <item>
      <widget class="adiscope::MenuAnim" name="rightMenu" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QGroupBox" name="config_panel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Configuration</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QTabWidget" name="tabWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">/* QTabsWidgets */
QTabWidget::pane {
  border-top: 0px;
}
QTabBar::tab {
  min-width: 100px;
  min-height: 32px;
  padding-bottom: 5px;
  font: bold;
}
QTabBar::tab:selected {
  color: white;
  border-bottom: 2px solid rgba(243, 109, 10);
  margin-top: 0px;
}
QTabBar::tab:!selected {
  border-bottom: 2px solid rgba(55, 55, 64);
  margin-top: 0px;
}</string>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <property name="usesScrollButtons">
              <bool>false</bool>
             </property>
             <widget class="QWidget" name="tabConstant">
              <attribute name="title">
               <string>Constant</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tabWaveform">
              <attribute name="title">
               <string>Waveform</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="styleSheet">
                  <string notr="true">padding: 10px;</string>
                 </property>
                 <property name="text">
                  <string>Waveform type</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="type">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>Sine</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Square</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Triangle</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Sawtooth</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <layout class="QGridLayout" name="waveformGrid"/>
               </item>
               <item>
                <spacer name="verticalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::MinimumExpanding</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tabBuffer">
              <attribute name="title">
               <string>Buffer</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QPushButton" name="load_file">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>64</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
height: 128px;
width: 128px;
border: none;
background-image: url(:/icons/floppy.svg);
background-position: center center;
background-repeat: no-repeat;
}
QPushButton:hover {
background-image: url(:/icons/floppy_hover.svg);
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_3">
                 <property name="title">
                  <string>File information</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_5">
                    <property name="font">
                     <font>
                      <family>DejaVu Sans</family>
                      <pointsize>10</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>File path:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLabel" name="label_path">
                    <property name="font">
                     <font>
                      <family>DejaVu Sans</family>
                      <pointsize>10</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_9">
                    <property name="font">
                     <font>
                      <family>DejaVu Sans</family>
                      <pointsize>10</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>Format:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLabel" name="label_format">
                    <property name="font">
                     <font>
                      <family>DejaVu Sans</family>
                      <pointsize>10</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_7">
                    <property name="font">
                     <font>
                      <family>DejaVu Sans</family>
                      <pointsize>10</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>Size:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLabel" name="label_size">
                    <property name="font">
                     <font>
                      <family>DejaVu Sans</family>
                      <pointsize>10</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tabMath">
              <attribute name="title">
               <string>Math</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="adiscope::Math" name="mathWidget" native="true">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="channelsList"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>adiscope::MenuAnim</class>
   <extends>QWidget</extends>
   <header>menu_anim.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>adiscope::Math</class>
   <extends>QWidget</extends>
   <header>math.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
